robotoff/app/api.py

```diff
@@ -1150,13 +1150,19 @@ def on_get(self, req: falcon.Request, resp: falcon.Response):
         count: int = req.get_param_as_int("count", min_value=1, default=25)
         question_type: str = req.get_param("type")
         value_tag: str = req.get_param("value_tag")
+        country: Optional[str] = req.get_param("country")
         server_domain: Optional[str] = req.get_param("server_domain")
 
+        import pdb
+
+        pdb.set_trace()
+
         query_parameters = {
             "keep_types": [question_type],
```

Comment:
Replace this by:
```suggestion
            "keep_types": [question_type] if question_type else None,
```

Reply:
That said, I don't think using this API without question_type as any sense, so I would better make question_type mandatory.